undertow/WFLY-11031_Formatted_Access_Logs.adoc

```diff
@@ -0,0 +1,70 @@
+= JSON Formatted Access Logs
+:author:            James Perkins
+:email:             jperkins@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Currently access logs can only be written as plain text. This plain text will be written to either a specific file or
+logged via the normal logging facilities. It's desirable for a log aggregator to accept formatted log messages. Using
+JSON seems to be the most common format.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-11031[WFLY-11031]
+* https://issues.jboss.org/browse/EAP7-1183[EAP7-1183]
+
+=== Related Issues
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:jstourac@redhat.com[Jan Stourac]
+
+=== Affected Projects or Components
+
+* WildFly
+** Undertow subsystem
+
+=== Other Interested Projects
+
+* OpenShift
+
+== Requirements
+
+=== Hard Requirements
+
+* The output must be well formed JSON.
+* The formatted message must be on a single line.
+* The output must be able to write to standard out. This is required for OpenShift.
```

Comment:
Is stdout all it can write to? The "setting=console-access-log" resource name implies that.

If that's all it can write to, perhaps add a Non-Requirement re writing to the access log file or piping output through the logging subsystem to whatever handlers are there.

Reply:
That's not a bad idea. I'll add that thanks!